bin/build_all

```diff
@@ -253,6 +253,7 @@ then
         docker_build $top_dir/docker/sawtooth-dev-python docker/ sawtooth-settings-tp
         docker_build $top_dir/docker/sawtooth-dev-python docker/ sawtooth-identity-tp
         docker_build $top_dir/docker/sawtooth-dev-python docker/ sawtooth-block-info-tp
+        docker_build $top_dir/docker/sawtooth-dev-python docker/ sawtooth-battleship-tp
```

Comment:
Add a line to the other `build_python()` below, just copy the block_info and change.

Reply:
Fixed

Reply:
Squash the change into this commit

Reply:
broken up and squashed